api/src/main/java/javax/enterprise/inject/spi/configurator/ObserverMethodConfigurator.java

```diff
@@ -149,26 +148,38 @@
     ObserverMethodConfigurator<T> priority(int priority);
 
     /**
-     * Define Consumer to call when event is notified
+     * Define an operation that accepts a context of a fired event.
      *
      * @param callback to call for the event notification
      * @return self
      */
-    ObserverMethodConfigurator<T> notifyWith(Consumer<T> callback);
+    ObserverMethodConfigurator<T> notifyWith(EventConsumer<T> callback);
 
     /**
-     * Defines a BiConsumer to call when event is notified
+     * Allows modification of the asynchronous status of the observer to build.
```

Comment:
This is also unrelated to CDI-612 (just formatting change,). Pls create a new JIRA issue.